Modules/_sqlite/statement.c

```diff
@@ -165,6 +161,9 @@ static int pysqlite_check_remaining_sql(const char* tail)
                 break;
             case ' ':
             case '\t':
+                if (state == COMMENTEND_1) {
+                    state = IN_COMMENT;
+                }
                 break;
             case '\n':
             case 13:
```

Comment:
Since we are here, how about change `13` to '\r' for better readability.

Reply:
+1